apps/builder/app/builder/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -574,15 +576,16 @@ export const CssValueInput = ({
   const isKeywordValue = value.type === "keyword" && hasItems;
   const suffixRef = useRef<HTMLDivElement | null>(null);
 
-  const suffix = (
-    <Box ref={suffixRef}>
-      {isUnitValue
-        ? unitSelectElement
-        : isKeywordValue
-        ? keywordButtonElement
-        : null}
-    </Box>
-  );
+  suffix =
+    suffix === false ? null : (
```

Comment:
suffix?: ReactNode

Now we have that 
false and undefined has different meaning?

Reply:
fixed